src/mixxx.cpp

```diff
@@ -842,6 +842,23 @@ void setVisibilityOptionState(QAction* pAction, ConfigKey key) {
     pAction->setChecked(pVisibilityControl != NULL ? pVisibilityControl->get() > 0.0 : false);
 }
 
+void MixxxMainWindow::toggleCheckedSamplers() {
+    m_pViewShowSamplers->blockSignals(true);
+    ConfigKey key("[Samplers]", "show_samplers");
+    m_pViewShowSamplers->setChecked(ControlObject::get(key));
+    m_pViewShowSamplers->blockSignals(false);
+}
+
+void MixxxMainWindow::linkSkinWidget(ControlObjectSlave*& pCOS, 
```

Comment:
we try to avoid non const references, because you cannot see that the value might be changed when calling the function. User a ControlObjectSlave** instead.